functests/vrf/vrf.go

```diff
@@ -244,3 +188,63 @@ func paramInParamList(param string, paramRange []string) error {
 	}
 	return fmt.Errorf("error: wrong parameter %v", param)
 }
+
+func defineDesсribeTable(node string, ipStack string, namespace string, apiclient *client.ClientSet, nodeName string, testNetworkBlue string, testNetworkRed string) {
```

Comment:
I think testNetworkBlue can be named in a better way.
Like, blueVRFNetworkName or something like that. Also, this is a generic function but you are already assuming the value in the parameter name.
Maybe it's better to switch to firstVRFNetwork, secondVRFNetwork or something along that line?

Reply:
I switched most of the var names to *firstVRF*/*secondVRF*, please review it again